src/clientversion.cpp

```diff
@@ -47,6 +47,11 @@ static std::string FormatVersion(int nVersion)
     return strprintf("%d.%d.%d", nVersion / 10000, (nVersion / 100) % 100, nVersion % 100);
 }
 
+std::string FormatVersion()
```

Comment:
Returns a terse string for reusability.
REF: https://github.com/bitcoin-core/gui/pull/140/commits/e0ff8e263dc96199385e31e481e64ad3a242fcd9#r536344384